index.html

```diff
@@ -0,0 +1,550 @@
+            Smashing Magazine itself! We’ll talk through the gains and ….
+          </p>
+        </div>
+
+        <div class="md">
+          <p>9 DAYS AGO</p>
+
+          <h3>
+            Smashing Podcast Episode 7 With Stephanie Stimac And Aaron
+            Gustafson: What’s New In Microsoft Edge?
+          </h3>
+
+          <p>By Drew McLellan</p>
+
+          <p>
+            We’re taking a look at the brand new version of Microsoft Edge and
+            its Chromium-based rendering engine. What does that change mean
+            for web developers, and how does the death of Microsoft’s own
+            rendering engine impact the web? Drew McLellan talks to …
+          </p>
+        </div>
+
+        <div class="md">
+          <p>10 DAYS AGO</p>
+
+          <h3>How To Design Powerful Narratives On Mobile</h3>
+
+          <p>By Suzanne Scacca</p>
+
+          <p>
+            Want to build a website or PWA that attracts a large audience and
+            then converts as much of that audience into paying customers as
+            possible? If that’s the case, what you need to do is use
+            storytelling in your design.
+          </p>
+        </div>
+
+        <div class="md">
+          <p>13 DAYS AGO</p>
+
+          <h3>How To Empower Design Teams By Measuring Value</h3>
+
+          <p>By Dave Cunningham</p>
+
+          <p>
+            How do we know if we are truly impactful as a design team? Are we
+            seen as a vehicle to deliver a solution that moves a needle? It’s
+            time to empower our design teams and give them one voice to show
+            how and when design really adds value.
+          </p>
+        </div>
+
+        <div class="lg">
+          <div class="lg-img"></div>
+
+          <h3>New on Smashing TV</h3>
+
+          <p>
+            A channel where seasoned developers show how they design, build,
+            work and play. Coming up next:
+          </p>
+
+          <button>See all upcoming episodes →</button>
+
+          <p>Already have an account? Sign in!</p>
+        </div>
+      </div>
+    </section>
+    <div class="latest-articles">
+      <p>SEE ALL LATEST ARTICLES</p>
+    </div>
+    <!-- pre launch -->
+    <section>
+      <h2>pre LAUNCH</h2>
+      <div class="pre-launch bg-gray-3">
+        <p>PRE LAUNCH FEBRUARY 2020</p>
+        <h3>Ethical Design Handbook:</h3>
+        <h3>Leaving Dark Patterns Behind.</h3>
+        <p>
+          WWith <strong>practical techniques & strategies</strong> to
+          influence a positive change in your company. Help <br />your
+          business grow sustainably — without dark patterns — and comply with
+          GDPR and CCPA.
+        </p>
+        <button>Learn more about the book</button>
+      </div>
+    </section>
+    <!-- community post -->
+    <section>
+      <h2>community post</h2>
+      <div class="community-post">
+        <div class="rotated"></div>
+        <div>
```

Comment:
line #268 should be `<article>` and not a `<div>`

Reply:
done